client/helpers/parts.ts

```diff
@@ -55,15 +74,16 @@ export interface GetPartRevisionBySuppliedIdReq extends BaseReq {
  *
  * @param args - The {@link CreatePartFromFileReq}.
  */
-export async function createPartFromFileIfNotExists({
```

Comment:
Deprecate the one below that returns `PartRevisionData` and instead return an interface of our own so we can add new things to it in the future in a non-breaking way.